doc/release-notes-bucash.md

```diff
@@ -32,162 +33,364 @@ If you are upgrading from a release older than 1.1.2.0, your UTXO database will
 to a new format. This step could take a variable amount of time that will depend
 on the performance of the hardware you are using.
 
-Other than that upgrading from a version lower than 1.5.0.1 your client is probably stuck
+Other than that upgrading from a version lower than 1.5.0.0 your client is probably stuck
 on a minority chain and need some manual intervention to make so that it follow the majority
 chain after the upgrade. For more detail please look at `reconsidermostworkchain` RPC commands.
 
-Main Changes in 1.5.1
+Main Changes in 1.6.0
 ---------------------
 
-- 10x transaction processing performance
-- Extended version message: [xversion](https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/doc/xversionmessage.md)
-- Adding a checkpoint at height 556767 both for the SV and the BCH chain
-
-Bug Fixed since 1.5.1.0-rc2
-------------------------
-
-- Documentation fixes
-- Make it possible to disable graphene and xthin preferential timer
-- Switch ctor on earlier in the init process if we are on the BCH chain
-- Improve Graphene decode rates by padding IBLT
-- Improve xversion QA testing
-- General improvements and fixes to the QA test framework
+- Add SCRIPT_ALLOW_SEGWIT_RECOVERY (May 15th upgrade)
+- Schnorr signature scheme (May 15th upgrade)
+- RPC enhancements
+- QA reliability improvemennts
+- Documentation improvements
+- Graphene now use CTOR
+- Graphene could use Fast Filter instead of Bloom Filter
+- Graphene: seeding of IBLT hash function
+- Graphene: use cheap hashes to avoid short id tx collision
+- Compact Block implementations (BIP 152)
+- Easier gitian build process (docker based)
+- Code restructure around the 3 block propagation techs we currently support (Graphene, CB, Xthin)
+- Move to C++14 and bump lib boost min ver to match
+- Update Windows build scripts
+- Simplify and update expedited code
+- Refactor Script interpreter as a "virtual machine" encapsulated by a class
+- implement xversion changeable key
 
```

Comment:
what about the electrum integration...aren't we waiting for that to be merged still?

Reply:
I don't think we should wait for it to release 1.6. I think we need as much run time as possible for 1.6. So the earlier we release the better. 

We could add [electrs](https://github.com/dagurval/electrs) integration in a subsequent release (e.g. 1.6.0.1)